core/assets/bundles/bundle.properties

```diff
@@ -1578,7 +1578,7 @@ lenum.sin = Sine, in degrees.
 lenum.cos = Cosine, in degrees.
 lenum.tan = Tangent, in degrees.
 #not a typo, look up 'range notation'
-lenum.rand = Random number in range [0, value).
+lenum.rand = Random number in range [0, value].
```

Comment:
**This was not a typo.** Read the comment above it.

Reply:
Oh...
Im so dumb...